### PR TITLE
Show testpaths option in the header if it has been used for collection

### LINKED ISSUE #4875: Show testpaths in pytest header when set
Include `testpaths` in the header, next to rootdir/inifile

_Originally posted by @blueyed in https://github.com/pytest-dev/pytest/issues/4873#issuecomment-468851390_

### PR DESCRIPTION
**Pull request by nicoddemus**
Fix #4875

### CODE REVIEW

File: changelog/4875.feature.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The `testpaths <https://docs.pytest.org/en/latest/reference.html#confval-testpaths>`__ configuration option is now displayed next
+to the ``rootdir`` and ``inifile`` lines in the pytest header if the option is in effect, i.e., directories or file names were
+explicitly passed in the command line.
```
Comment (blueyed) [inline changelog/4875.feature.rst]:
```suggestion
not explicitly passed in the command line.
```

**Comment by nicoddemus** [comment]
Thanks for the quick review.

I've pushed another commit also removing `inifile:` from the header if we don't have an `inifile`, I think it makes the output leaner.

File: testing/test_terminal.py [hunk 2]
```diff
@@ -1199,15 +1219,6 @@ def test_summary_stats(exp_line, exp_color, stats_arg):
     assert color == exp_color
 
 
-def test_no_trailing_whitespace_after_inifile_word(testdir):
```
Comment (nicoddemus) [inline testing/test_terminal.py]: This test is now unnecessary, `test_header` already covers this.

Comment (blueyed) [reply testing/test_terminal.py]: Great.

**Pull request merged**